src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,21 @@
+import kotlinx.cli.*
+import java.net.URL
+
+fun fetchBlocks(startHeight: Int): String {
+    val url = URL("https://mempool.space/api/blocks/$startHeight")
+    return url.readText()
```

Comment:
You need to handle errors as well: what happens if `mempool.space` returns an HTTP 500 for example?